src/pages/index.mdx

```diff
@@ -12,17 +8,260 @@ export const disableEditGithub = true
 
 <HomeHeader
   title="Nitric Documentation"
-  description="Learn how to quickly get started with Nitric by exploring tutorials, APIs, and platform resources. Discover how Nitric can empower you to build applications efficiently and effectively."
+  description="Build cloud backends that run anywhere, fast."
 />
 
-<GettingStarted />
+Nitric is a cloud framework with common resources like APIs, webhooks, databases, queues, topics, buckets, and more.
 
-<Concepts />
+However, Nitric is unique - it doesn't just deploy the resources, it helps you interact with them. It also makes them plugable, so you can swap services or even whole clouds without changing your code.
 
-{/* <BuildingBlocks /> */}
+It's what's missing between applications and infrastructure automation.
 
-<Libraries />
+Oh, and it supports basically any language, like JavaScript, TypeScript, Python, Go, you name it.
 
-<Providers />
+## Services
 
-<Community />
+Services are the heart of nitric apps, they're the entrypoints to your code. They can serve as APIs, webhooks, schedule handlers, subscribers and a lot more. You create services by telling Nitric where to look for your code and how to run it.
+
+```yaml {{ label:"nitric.yaml" }}
+name: example
+services:
+    - match: services/*.ts
+      start: npm run dev:services $SERVICE_PATH
+```
+
+You might have one service that handles everything, or a service for each route. It's up to you.
+
+## Resources
+
+Resources are the building blocks of your apps. They're the databases, queues, buckets, etc. that your services interact with. If you want a resource, you just ask for it.
+
+<CodeGroup>
+
+```javascript {{ label:"services/example.js" }}
+import { bucket } from '@nitric/sdk'
+
+const profiles = bucket('profiles').allow('read', 'write', 'delete')
+```
+
+```python {{ label:"services/example.py" }}
+from nitric.resources import bucket
+from nitric.application import Nitric
+
+profiles = bucket('profiles').allow('read', 'write', 'delete')
+
+Nitric.run()
+```
+
+```go {{ label:"services/example/main.go" }}
+import (
+	"fmt"
+
+	"github.com/nitrictech/go-sdk/nitric"
+)
+
+func main() {
+	profiles, _ := nitric.NewBucket("profiles").Allow(nitric.BucketRead, nitric.BucketWrite, nitric.BucketDelete)
+
+	if err := nitric.Run(); err != nil {
+		fmt.Println(err)
+	}
+}
+```
+
+```dart {{ label:"services/example.dart" }}
+import 'package:nitric_sdk/nitric.dart';
+
+final profiles = Nitric.bucket("profiles").allow([
+  BucketPermission.read,
+  BucketPermission.write,
+  BucketPermission.delete,
+]);
```

Comment:
```suggestion
void main() {
  final profiles = Nitric.bucket("profiles").allow([
    BucketPermission.read,
    BucketPermission.write,
    BucketPermission.delete,
  ]);
}
```

Reply:
Previous examples of Dart don't show the main method, just suggesting it's added as Go shows their main method.